src/pages/search-result/search-result.js

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import { Header, VideoThumbnail } from '../../components';
+import { StyledResultsDiv } from './styled';
+import { fetchSearchResults } from '../../api';
+
+class SearchResultPage extends PureComponent {
+  state = {
+    results: { items: [] },
+  };
+
+  async componentDidMount() {
+    const { query } = this.props.match.params;
+    const results = await fetchSearchResults(query);
+    this.setState({ results: await results.data });
+  }
+
+  render() {
+    const { results } = this.state;
+    return (
+      <div>
+        <Header />
+        <StyledResultsDiv>
+          {results.items.map(({ snippet, id }) => (
```

Comment:
Add a key for each item of the list
Explore why are keys required for list by React. What and Why does React throw warning